integration-tests/deployment/ccip/test_assertions.go

```diff
@@ -72,7 +72,15 @@ func ConfirmCommitWithExpectedSeqNumRange(
 	}
 
 	defer subscription.Unsubscribe()
-	timer := time.NewTimer(5 * time.Minute)
+	var duration time.Duration
+	deadline, ok := t.Deadline()
+	if ok {
+		// add a minute to the deadline to ensure we don't miss it
```

Comment:
Not sure you can tweak a deadline this way - I believe the Go test runner will just kill the test once its past the deadline. Is this what you want?

Reply:
Ah I think this comment is a bit misleading, maybe "make the timer end a minute before the deadline so we don't miss it"